checkstyle.xml

```diff
@@ -66,7 +66,9 @@
         <module name="IllegalImport" />
         <!-- defaults to sun.* packages -->
         <module name="RedundantImport" />
-        <module name="UnusedImports" />
+        <module name="UnusedImports">
+            <property name="processJavadoc" value="true"/>

```

Comment:
This should make checkstyle to process javadocs and not fail every time we mention an unused method in the comment.